girder/test/test_access.py

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###############################################################################
+#  Copyright 2014 Kitware Inc.
```

Comment:
2017

Reply:
Are we allowed to just remove the date? If so I would do it across Girder so we don't have to have sporadic commits changing these values each year.